index.html

```diff
@@ -192,12 +192,12 @@ <h2>How do you determine: Fast, Average, Slow?</h2>
 
           <ul>
             <li>
-              “fast” TTFB &lt; 200 ms is based on the <a href="https://developers.google.com/speed/docs/insights/Server">server responsiveness</a>
+              “fast” TTFB &lt; 500 ms is based on the <a href="https://developers.google.com/speed/docs/insights/Server">server responsiveness</a>
                    best practice.
             </li>
             <li>
-              “slow” TTFB &gt;= 1000 ms is based on the requirement for the fast First Contentful Paint (FCP) threshold
-              to be &lt; 1000 ms. A server response that exceeds 1000 ms cannot, by definition, meet the fast FCP goal.
+              “slow” TTFB &gt;= 1500 ms is based on the requirement for the fast First Contentful Paint (FCP) threshold
+              to be &lt; 1800 ms. A server response that exceeds 1500 ms will not, in most cases, meet the fast FCP goal.
```

Comment:
Can adjust the text here if you like - but slow TTFB is now 1500ms and fast FCP 1800ms, so the explanation needs to be tweaked a bit